src/OctoshiftCLI.Tests/gei/Handlers/MigrateRepoCommandHandlerTests.cs

```diff
@@ -1097,8 +1112,11 @@ public async Task Does_Not_Pass_Lock_Repos_To_StartMigration_For_GHES()
 
             _mockTargetGithubApi.Setup(x => x.DoesOrgExist(TARGET_ORG).Result).Returns(true);
 
-            _mockHttpDownloadService.Setup(x => x.DownloadToBytes(It.IsAny<string>()).Result).Returns(Array.Empty<byte>());
-            _mockAzureApi.Setup(x => x.UploadToBlob(It.IsAny<string>(), It.IsAny<byte[]>()).Result).Returns(new Uri("https://example.com/resource"));
+
+            _mockHttpDownloadService.Setup(x => x.DownloadToFile(It.IsAny<string>(), It.IsAny<string>()));
+            _mockFileSystemProvider.Setup(x => x.OpenRead(It.IsAny<string>())).Returns(It.IsAny<FileStream>);
```

Comment:
same

Reply:
Addressed in 7d436e04071bf868990e1d32d6276668c1db1a35 :tada: